### ISSUE TITLE
Batch size with Zero-Shot Evaluation

### ISSUE THREAD

**Issue by fealty94** [princeton-vl/WAFT#18]
Dear Authors

I am writing to express my appreciation for your inspiring work on WAFT. While I was attempting to reproduce the results mentioned in your paper, I noticed a slight discrepancy between the manuscript and the provided GitHub repository.

In Section 5.2 (page 7), it is stated that the model was trained with a batch size of 32, a learning rate of $10^{-4}$, steps 50000.

However, in the chairs-things.json config file on your GitHub, the values are set to "batch_size": 30, "lr": 4e-4, and steps 100000.
(https://github.com/princeton-vl/WAFT/blob/waftv2/config/a2/twins/chairs-things.json)

Could you please clarify which hyperparameters were used to achieve the results reported in the paper?

Additionally, if the batch size of 30 was distributed across multiple GPUs (e.g., 8 GPUs), I would appreciate it if you could confirm the actual global batch size used during training. Because if the batch size of 30 is the case, 30 // 8 == 3, 3*8 = 24 so actual batch size is 24.

According to the DataLoader setup in your code:
train_loader = data.DataLoader(..., batch_size=args.batch_size // num_gpu, ...)

If the training was conducted using 8 GPUs with args.batch_size set to 30, the local batch size per GPU would be 3 (30 // 8 == 3). This implies that the actual global batch size used for the gradient update would be 24 ($3 \times 8 = 24$), effectively dropping 6 samples per step.

Could you please clarify if the reported results were achieved with a global batch size of 32, 30, or 24? I want to ensure that my reproduction setup, including gradient accumulation, precisely matches your original experiments.

Thank you for your time and for sharing your valuable research with the community.

**Comment by MemorySlices** [comment]
Sorry about the confusion! This is typo - there was a version trained with batch size 30 on 10 A6000 GPUs, which provides similar results to batch size 32 on 8 L40 GPUs.
